src/diffpy/pdffit2/tests/run.py

```diff
@@ -1,34 +1,38 @@
 #!/usr/bin/env python
 ##############################################################################
 #
-# (c) 2024 The Trustees of Columbia University in the City of New York.
-# All rights reserved.
+# diffpy.pdffit2    by DANSE Diffraction group
+#                   Simon J. L. Billinge
+#                   (c) 2012 Trustees of the Columbia University
+#                   in the City of New York.  All rights reserved.
 #
-# File coded by: Billinge Group members and community contributors.
+# File coded by:    Pavol Juhas
 #
-# See GitHub contributions for a more detailed list of contributors.
-# https://github.com/diffpy/diffpy.pdffit2/graphs/contributors
-#
-# See LICENSE.rst for license information.
+# See AUTHORS.txt for a list of people who contributed.
+# See LICENSE.txt for license information.
 #
 ##############################################################################
+
 """Convenience module for executing all unit tests with
+
 python -m diffpy.pdffit2.tests.run
 """
 
-import sys
-
-import pytest
 
 if __name__ == "__main__":
-    # show output results from every test function
-    args = ["-v"]
-    # show the message output for skipped and expected failure tests
-    if len(sys.argv) > 1:
-        args.extend(sys.argv[1:])
-    print("pytest arguments: {}".format(args))
-    # call pytest and exit with the return code from pytest
-    exit_res = pytest.main(args)
-    sys.exit(exit_res)
```

Comment:
keep the cookiecutter code but comment it out.  Later we want to migrate over to the cookiecutter way of running the tests if we can.